### DISCUSSION TITLE
[DOC] Example for Vaultwarden from me for the DOCs

### DISCUSSION THREAD

**Discussion by MrCybertux** [bunkerity/bunkerweb#1801] [Technical]
### Description

Since Vaultwarden is a awesome Password manager and i have it self hosted i wanted to share my example setup.
There are some pitfalls so i hope my example can help avoid these 🙂

### Proposed solution (optional)

Here is the example compose file i made (as TXT since Github blocks yml):
[docker-compose.txt](https://github.com/user-attachments/files/18094516/docker-compose.txt)

and my Modsec file placed under path/to/docker-vol/bw-data/configs/modsec-crs:
[vault.txt](https://github.com/user-attachments/files/18094536/vault.txt)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by TheophileDiot** [comment]
Hi @MrCybertux, just remembered today that Discussions exists on GitHub, could you open a P/R with the added files please ? Thank you :pray:
